ui/src/setupProxy.js

```diff
@@ -0,0 +1,16 @@
+const {createProxyMiddleware} = require('http-proxy-middleware');
+
+module.exports = function (app) {
+  app.use(
+    createProxyMiddleware(['/auth', '/user'], {
+      target: 'http://localhost:4200',
```

Comment:
What will happen in case of production ?

Reply:
I think we can use the env value of `REDIRECT_URI` instead of hard coding this value

Reply:
yes ,we should get it from env